docs/functional_architecture.rst

```diff
@@ -0,0 +1,70 @@
+.. _doc_functional_architecture:
+
+=========================================
+WiP : Parsec client functional components
+=========================================
+
+Missing translations and figures
+
+Mountpoint
+**********
+Le composant mountpoint est responsable de l’interaction avec le système de fichiers pour
+communiquer avec les applications tiers selon une logique très simple consistant à transmettre toutes les
+requêtes natives au système de fichiers virtuel défini dans le composant core.
+
+GUI
+***
+Le composant gui gère l'interface utilisateur interagit avec le composant core ou directement avec le
+système de fichiers natif, mais pas avec le composant mountpoint. Contrairement au composant mountpoint,
+le gui souscrit aux événements exposés par le serveur de métadonnées, tels que les notifications en cas
+d’actions concurrentes, de suppression d'un groupe etc. cela afin d’en informer l’utilisateur.
+
+CORE
+****
+Outre les deux fonctions précédentes, le composant core intègre toute la logique du client et contient
+cinq sous-modules (Figure 3) dont les rôles sont les suivants :
+
+● Le Système de Fichiers Virtuels (VFS) reçoit toutes les requêtes relatives au système de fichiers en
+provenance du composant mountpoint et, à cette fin, dispose d’une API qui simule une interface de
+système de fichiers. Pour optimiser ses performances, ce composant ne cherche pas à pousser les
+modifications jusqu’au serveur de métadonnées; il se contente de stocker de manière chiffrées les
+modifications sur le disque dur de la machine locale.
+
+● Le Synchroniseur est le composant qui transfère périodiquement les données modifiées stockées sur la
+machine locale vers le serveur de métadonnées. Il s’occupe d’écouter les notifications du serveur de
+métadonnées en cas de modification des données par un autre terminal ainsi que de résoudre les conflits
+de version entre les données locales et celles du serveur de métadonnées.
+
+● Le Gestionnaire d’Identités stocke dans la mémoire locale l’identité de l’utilisateur connecté (sous la
+forme d’une session). La passphrase de l’utilisateur, qui n’est pas stockée, à laquelle on a rajouté un sel,
```

Comment:
Does `passphrase` need to be translated ?

Reply:
This page is a WiP and coming as is from the "Cible de sécurité Parsec", I'm closing it then

Reply:
Actually changed it just in case